_includes/post-gallery.html

```
@@ -0,0 +1,55 @@
+
+{% assign gallery_data = page.galleries[include.gallery] %}
+{% if gallery_data and gallery_data.images.size > 0 %}
+<div class="post-gallery light-gallery-wrapper mb-30">
+  {% if gallery_data.caption %}
+  <p class="text-center text-muted mb-10"><small>{{ gallery_data.caption }}</small></p>
+  {% endif %}
+  <div class="tiles tiles-s">
+    <div class="items row">
+      {% assign image_count = gallery_data.images | size %}
+      {% for item in gallery_data.images %}
+        {% if image_count <= 2 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)